src/empty-states/src/EmptyState.js

```diff
@@ -0,0 +1,184 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { Button } from '../../buttons'
+import { Pane } from '../../layers'
+import { majorScale, minorScale } from '../../scales'
+import { useTheme } from '../../theme'
+import { Link, Heading, Paragraph } from '../../typography'
+
+/* eslint-disable react/prop-types */
+
+const HorizontalOrientation = ({
```

Comment:
Let's memoize these components to prevent unnecessary re-renders. 

You can do: 

```
import React, { memo } from 'react' 
```

and then `const HoriztontalOrientation = memo(function HorizontalOrientation({ ... }) { return ... })`